database/gormdb_test.go

```diff
@@ -185,41 +184,43 @@ func TestGormDBGetCourses(t *testing.T) {
 	qtest.CreateCourse(t, db, admin, c2)
 	qtest.CreateCourse(t, db, admin, c3)
 
-	courses, err := db.GetCourses()
+	gotCourses, err := db.GetCourses()
 	if err != nil {
 		t.Fatal(err)
 	}
 	wantCourses := []*pb.Course{c1, c2, c3}
-	if !reflect.DeepEqual(courses, wantCourses) {
-		t.Errorf("have %v want %v", courses, wantCourses)
+	if diff := cmp.Diff(wantCourses, gotCourses, protocmp.Transform()); diff != "" {
+		t.Errorf("GetCourses() mismatch (-wantUser, +gotUser):n%s", diff)
 	}
+
 	// An empty list should return the same as no argument, it makes no
 	// sense to ask the database to return no courses.
-	coursesNoArg, err := db.GetCourses([]uint64{}...)
+	gotCourses, err = db.GetCourses([]uint64{}...)
 	if err != nil {
 		t.Fatal(err)
 	}
-	if !reflect.DeepEqual(coursesNoArg, wantCourses) {
-		t.Errorf("have %v want %v", coursesNoArg, wantCourses)
+	if diff := cmp.Diff(wantCourses, gotCourses, protocmp.Transform()); diff != "" {
+		t.Errorf("GetCourses() mismatch (-wantUser, +gotUser):n%s", diff)
 	}
 
-	course1, err := db.GetCourses(c1.ID)
+	gotCourse, err := db.GetCourses(c1.ID)
 	if err != nil {
 		t.Fatal(err)
 	}
-	wantCourse1 := []*pb.Course{c1}
-	if !reflect.DeepEqual(course1, wantCourse1) {
-		t.Errorf("have %v want %v", course1, wantCourse1)
+	wantCourse := []*pb.Course{c1}
+	if diff := cmp.Diff(wantCourse, gotCourse, protocmp.Transform()); diff != "" {
+		t.Errorf("GetCourses() mismatch (-wantUser, +gotUser):n%s", diff)
 	}
 
-	course1and2, err := db.GetCourses(c1.ID, c2.ID)
+	gotCourse, err = db.GetCourses(c1.ID, c2.ID)
 	if err != nil {
 		t.Fatal(err)
 	}
-	wantCourse1and2 := []*pb.Course{c1, c2}
-	if !reflect.DeepEqual(course1and2, wantCourse1and2) {
-		t.Errorf("have %v want %v", course1and2, wantCourse1and2)
+	wantCourse = []*pb.Course{c1, c2}
+	if diff := cmp.Diff(wantCourse, gotCourse, protocmp.Transform()); diff != "" {
+		t.Errorf("GetCourses() mismatch (-wantUser, +gotUser):n%s", diff)
```

Comment:
Should be `-wantCourses +gotCourses` here. Also, the variable names should be similarly updated.